pages/user/moderation.js

```diff
@@ -2028,7 +2030,35 @@ const UserModerationQueue = ({
                   </a>
                 </span>
                 <span className="col-email text-muted">{profile.content.preferredEmail}</span>
-                <span className="col-created">{formatDateTime(profile.tcdate)}</span>
+                <span className="col-created">
+                  <span
+                    className={`${profile.tmdate < twoWeeksAgo ? 'passed-moderation' : ''}`}
+                  >
+                    {formatDateTime(profile.tmdate, {
+                      day: '2-digit',
+                      month: 'short',
+                      year: '2-digit',
+                      hour: '2-digit',
+                      minute: '2-digit',
+                      second: undefined,
+                      timeZoneName: undefined,
+                      hour12: false,
+                    })}
+                  </span>
+                  /
+                  <span>
+                    {formatDateTime(profile.tcdate, {
```

Comment:
What about only displaying 1 date if the dates are the same? It would make it much more obvious which profiles have different created/modified dates.